picoquic/frames.c

```diff
@@ -598,25 +665,23 @@ int picoquic_stream_network_input(picoquic_cnx_t* cnx, uint64_t stream_id,
     return ret;
 }
 
-int picoquic_decode_stream_frame(picoquic_cnx_t* cnx, uint8_t* bytes,
-    size_t bytes_max, size_t* consumed, uint64_t current_time)
+uint8_t* picoquic_decode_stream_frame(picoquic_cnx_t* cnx, uint8_t* bytes, const uint8_t* bytes_max, uint64_t current_time)
 {
-    int ret;
-    int fin = 0;
     uint64_t stream_id;
-    size_t data_length;
+    size_t   data_length;
     uint64_t offset;
+    int      fin;
+    size_t   consumed;
 
```

Comment:
For code formatting, we have a specific cmake target. See PR #66 for details, but the idea is to use clang-format to automatically reset indents, comments, block ends, etc. we should consider running that after the current series of updates.